chapters/en/slides/module3/module3_03.Rmd

```diff
@@ -0,0 +1,245 @@
+(but we know there are some since the axis extends that far).
+
+---
+
+## Many distributions can't be effectively compared with densities either
+
+```{python}
+(alt.Chart(movies_extended).mark_area().transform_density(
+    'Worldwide Gross',
+    groupby=['Major Genre'],
+    as_=['Worldwide Gross', 'density'])
+ .encode(
+    alt.X('Worldwide Gross'),
+    alt.Y('density:Q'),
+    alt.Color('Major Genre')))
+```
+
+Notes:
+Although we saw previously that a layered density chart
+is better than a stacked histogram,
+it is no effective in comparing this many distributions
+and has many of the same problems as the histogram in the previous slide.
+
+So what can we do to create an effective comparison between all the genres?
+
+---
+
+## Bar charts are effective for comparing a single value but hides any variation
+
+```{python}
+alt.Chart(movies_extended).mark_bar().encode(
+    alt.X('mean(Worldwide Gross)'),
+    alt.Y('Major Genre'))
+```
+
+Notes:
+We could use a barplot if we were only interested in comparing a single value
+from each distribution,
+such as the mean or median.
+However,
+this is generally not a good idea because any plot that only shows a single value
+will hide the variation in our data
+and could lead to us drawing the wrong conclusion
+as you will see in the next slide.
+
+---
+
+## Showing a single value can lead to incorrect conclusions
+
+<img src=/module3/barplot-hiding-points.png></img>
+
+[Beyond Bar and Line Graphs: Time for a New Data Presentation Paradigm](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002128)
+
+Notes:
+This image illustrates how the same barplot (A)
+could have been generated from several different distributions (B, C, D, or E).
+If we just looked at the barplot,
+we could not tell which of the distributions it came from,
+but looking at the points directly
+we would arrive at widely different conclusion to what was going on with our data.
+
+This is why we most of the times need to present a richer representation of the data
+such as a histogram or a density plot.
+Since these do not work well for multiple distributions,
+we will use a compromise: the boxplot.
+
+---
+
+## Boxplots show several key statistic from a distribution
+
+<img src=/module3/boxplot-schematic.webp></img>
+
+[What does a box plot tell you? Simply psychology](https://www.simplypsychology.org/boxplots.html)
+
+Notes:
+Instead of showing just the mean or median,
+a box plot shows 5 values.
+The box always represents the median and the upper and lower quartile,
+which means that 50% of the data points lie within the range of the box.
+The whiskers can represent a few different things,
+in this picture they re the min and min and the max (the range) of our data,
+but they can also be 1.5x the interquartile range
+and then any observations outside this are drawn out as individual points
+(this is the case in Altair).
+
+---
+
+## Boxplots can effectively compare multiple distrubtions
+
+```{python}
```

Comment:
Can we show the bar chart and the bloxplot here side-by-side and compare and contrast them? I think that would really bring the message home. I think we should do a lot of this in this course, it's quite effective and illustrative. As it is hard to hold both images in your head at once.

Reply:
Agree its good to learn by contrast and will try to include this! I just needto figure out the space constraints and the fact that we weren't planning to teach layouts until lecture 7. We could either teach it earlier and make it on a separate slide.

Reply:
Added, not sure how how intuitive it is to introduce the layout options here, but lets try!